src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -876,7 +876,22 @@ private void testDeleteQuestionAction(int qnNumber) throws MaximumRetriesExceede
         feedbackEditPage.waitForConfirmationModalAndClickOk();
         feedbackEditPage.verifyStatus(Const.StatusMessages.FEEDBACK_QUESTION_DELETED);
         assertNull(getFeedbackQuestion(courseId, feedbackSessionName, qnNumber));
+    }
+
+    private void testEditWithEmptyQuestionTextThenDeleteQuestionAction(int qnNumber)
+            throws MaximumRetriesExceededException {
+        ______TS("qn " + qnNumber + " edit the question with invalid input then delete question");
+
+        feedbackEditPage.clickEditQuestionButton(qnNumber);
+        feedbackEditPage.fillQuestionTextBox("", qnNumber);
+        feedbackEditPage.clickSaveExistingQuestionButton(qnNumber);
+        String expectedString = Const.StatusMessages.FEEDBACK_QUESTION_TEXTINVALID;
```

Comment:
Just a query: Why not rename this to use underscores as well?

And can't we just inline `expectedString`, doesn't seem to be useful to have a separate local variable here?

Reply:
Opps, that's just because previous I did not use Const. Yes I feel inline is better